ceph-dev-setup/build/build

```diff
@@ -35,8 +35,9 @@ rm -rf release
 # because autogen+configure will check for dependencies, we are forced to install them
 # and ensure they are present in the current host
 if [ -x install-deps.sh ]; then
-  echo "Ensuring dependencies are installed"
-  ./install-deps.sh
+    echo "Ensuring dependencies are installed"
+    [ "${FLAVOR}" == "crimson" ] && WITH_SEASTAR=1
+    WITH_SEASTAR=$WITH_SEASTAR ./install-deps.sh
```

Comment:
is it fine if `$WITH_SEASTAR` is undefined?

Reply:
it's fine.

in `install-deps.sh`, we have:
```sh
[ $WITH_SEASTAR ] && with_seastar=true || with_seastar=false
```